Xamarin.Essentials/Screenshot/Screenshot.ios.tvos.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Foundation;
+using UIKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Screenshot
+    {
+        internal static bool PlatformCanCapture => UIScreen.MainScreen != null;
+
+        static string GetTempFileName()
```

Comment:
See my comment on Android - we should probably use the Cache directory so the OS can clean up after us.